__tests__/app.test.js

```diff
@@ -56,6 +55,24 @@ describe('testing GET /api/articles/:article_id', () => {
         .expect(200);
     });
     test('should respond with an article object based on the given id', () => { 
+        const expected = {article : [{   
+            article_id: 2,
+            title: "Sony Vaio; or, The Laptop",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.",
+            created_at: "2020-10-16T05:03:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+          }]}
+
+        return request(app).get("/api/articles/2")
+        .then(data => {
+            expect(data._body).toMatchObject(expected);
+        })
+    });
```

Comment:
Is there meant to be a `comment_count` property on here? It looks like you're re-submitting task 12 but I can't see the comment count on here.

Reply:
niamh asked me to make 2 separate tests - one with the original / unchanged, and another for this new requirement given in this question